### PR TITLE
adding test for rendering tasks

### PR DESCRIPTION
**Pull request by emyjamalian**

### CODE REVIEW

File: components/TaskList/TaskList.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { Checkbox, ListItem, UnorderedList } from "@chakra-ui/react";
+
+const fruits = [
+  { id: 1, name: "Bananas" },
+  { id: 2, name: "Apples" },
+  { id: 3, name: "Strawberries" },
+  { id: 4, name: "Grapes" },
+  { id: 5, name: "Oranges" },
+];
+function FruitList() {
```
Comment (uetrozi) [inline components/TaskList/TaskList.test.js]: But if we don't import TasksList directly in order to test it, we aren't really testing the compoment, aren't we?

**Review by uetrozi — COMMENTED** [review]
Gotta change fruitList to TasksList. :)

File: components/TaskList/TaskList.test.js (hunk 1 above)
Comment (emyjamalian) [reply components/TaskList/TaskList.test.js]: Exactly but how can I render a component when its data is coming from DB?

Comment (uetrozi) [reply components/TaskList/TaskList.test.js]: ah, good question. Can you import TaskList but still feed it with data "manually"? Or testing input and task List together?

**Pull request merged**